src/App.tsx

```diff
@@ -1,26 +1,51 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import { useEffect, useState } from 'react';
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+import { todosSlice } from './features/todos';
+import { useAppDispatch, useAppSelector } from './app/hooks';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App = () => {
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState('');
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const dispatch = useAppDispatch();
+  const setTodos = (todos: Todo[]) =>
+    dispatch(todosSlice.actions.setTodos(todos));
 
-          <div className="block">
-            <TodoFilter />
-          </div>
+  useEffect(() => {
+    setLoading(true);
+    getTodos()
+      .then(data => {
+        setTodos(data);
+      })
+      .catch(() => {
+        setError('Something went wrong');
+      })
+      .finally(() => setLoading(false));
+  }, []);
```

Comment:
Fix it 
<img width="979" alt="image" src="https://github.com/mate-academy/react_redux-list-of-todos/assets/64750958/61adcacc-466d-44fe-8484-d567c22b5ee6">